homeassistant/components/tuya/strings.json

```diff
@@ -523,6 +523,9 @@
       "phase_c_voltage": {
         "name": "Phase C voltage"
       },
+      "add_ele": {
+        "name": "Current consumption"
+      },
```

Comment:
This description is rather unclear, what is it consuming?

Reply:
Please check the above conversation with [elupus](https://github.com/elupus) for the naming.

Reply:
Sorry, this just isn't correct English. This is an energy sensor (considering it is kWh).

Even with the discussion you reference above, it is completely unclear what this even is.

Energy is a measurement of power usage over time. As there is a time element, there is no such thing as "current energy". "Current consumption" in that way, doesn't make any sense at all. Especially considering current is often expressed in Ampere when we are talking electronics. Making this translation extremely confusing and not explanatory at all.

Reply:
The unit is ``kWh``. Under normal network conditions, Wi-Fi reports incrementally, and the individual values should be relatively small, such as 0.05 or 0.15. On the other hand, Zigbee reports the complete data, and the reported values gradually increase over time. You can check the example below for better understanding?
![example](https://github.com/home-assistant/core/assets/907831/b88f3b1d-c2a6-4315-b080-6c0dcec933b4)

Reply:
> The unit is kWh. Under normal network conditions, Wi-Fi reports incrementally, and the individual values should be relatively small, such as 0.05 or 0.15.

So, the values are `0.05, the next could be `0.15`, and the one after that could be `0.10` right?

Reply:
Yes, your understanding is correct. 0.05, next 0.15, then 0.10. So maybe "Real-time consumption" is better or any naming suggestions from your side?

Reply:
@frenck Hello, any updates for this PR? Should we use the new  "Real-time consumption" name or we should just remove this sensor entity update? Our customer want to present this sensor data.

Reply:
Remove " add_ele" updates?